laravel/database/seeds/ScheduleDataStaticSeeder.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Models\ScheduleDataStatic;
+
+class ScheduleDataStaticSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $allStaticData = [
+            ["MRI", "MRI", "MRI Procedure"],
+            ["ECT", "ECT", "ECT Procedure"],
+            ["Endoscopy", "Endo5", "Endoscopy Procedure"],
+            ["IR", "IR2", "IR Procedure"],
+            ["Pulmonary", "Pulmonary", "Bronchoscopy Procedure"],
+        ];
+        foreach($allStaticData as $data){
+            ScheduleDataStatic::insert([
```

Comment:
in the interest of avoiding duplicates (or worse, failure) if this seeder is run on an already populated database, what do you think about using [`firstOrCreate`](https://laravel.com/docs/8.x/eloquent#retrieving-or-creating-models) instead of `insert`?